### ISSUE TITLE
Flicker ARTIC.LVL around 23,254

### ISSUE THREAD

**Issue by cjritola** [jtrfp/terminal-recall#111]
ARTIC, sect. (23,254), terrain flickers severely and console blurts "PORTAL ENTRANCE [DE]ACTIVATED." Suspecting that the moving terrain hack is causing confusion.

**Comment by WDLmaster** [comment]
Strange, I cannot confirm that. I crossed the border a dozen times trying to recreate the bug but I found absolutely nothing wrong with the terrain. The only thing that appeared once was a missing cloud polygon for a second or so but nothing more. Why is it ARTIC3.LVL in particular? I directly selected that level. Maybe I have to play it in order (starting at ARTIC.LVL) for the bug to appear after the 2nd loading.
